src/darwin/Framework/CHIP/MTRDevice+XPC.h

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
Please remove this line.

Reply:
i think this is part of the project new file template so we should probably also fix it there